automation/check-patch.sh

```diff
@@ -29,8 +36,7 @@ if ! code_changed; then
 fi
 
 # Style and unit tests
-# required for the formatting
-pip install yapf
+pip install -r tests-requires.txt
```

Comment:
depends on your setup, if you are running in a virtualenv yes, it will work without root, but this is meant to be run with mock_runner.sh, and in that case, it's always running as root